tests/internal/unit/CMakeLists.txt

```diff
@@ -151,3 +151,24 @@ message(
 "Tests passed for `blt_split_source_list_by_language`.\n"
 "*****************************************************")
 
+message(STATUS "Checking for blt_check_code_compiles")
+
+blt_check_code_compiles(CODE_COMPILES  _hello_world_compiled
+                        VERBOSE_OUTPUT ON
+                        SOURCE_STRING
+"#include <iostream>
+
+int main(int, char**)
+{
+
+    std::cout << \"Hello World!\" << std::endl;
+
+    return 0;
+}
+")
```

Comment:
Suggestion: Should we use a "bracket argument" (multiline string) instead of a single quote string?
One benefit would be that we wouldn't need to modify the quotation marks
e,g,:

```cmake
[=[
#include <iostream>

int main(int, char**)
{
    std::cout << \"Hello World!\" << std::endl;
    return 0;
}
]=]
```
See: https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-argument

Reply:
This worked.. added it as a code snippet in the documentation as well.